server/app/views/admin/programs/ProgramImageView.java

```diff
@@ -135,7 +139,22 @@ public Content render(
     DivTag formsAndCurrentCardContainer =
         div().withClasses("grid", "grid-cols-2", "gap-10", "w-full");
     formsAndCurrentCardContainer.with(formsContainer);
-    formsAndCurrentCardContainer.with(renderCurrentProgramCard(request, programDefinition));
+
+    if (settingsManifest.getNorthStarApplicantUi(request)) {
+      // TODO(#7266): If the admin UI uses Thymeleaf, we can directly embed North Star Thymeleaf
+      // fragments without using HTMX.
```

Comment:
(question) Why not just render the thymeleaf to a string and add it as raw with j2html? Too many dependencies?